arangod/Scheduler/SupervisedScheduler.h

```diff
@@ -53,6 +54,10 @@ class SupervisedScheduler final : public Scheduler {
   void toVelocyPack(velocypack::Builder&) const override;
   Scheduler::QueueStatistics queueStatistics() const override;
 
+  Gauge<uint64_t>& ongoingLowPriorityTasks();
```

Comment:
i have concerns about this API, we now hand out references to Memory, that is not even under control of this class, but owned by some other place (we hold a reference here ourselves)

I would prefer to have an API that allows increment / decrement operations and can encapsulate shutdown checks, as in this situation the reference we are holding here might be invalid already, depending on the cleanup state in Metrics feature. For now this is fine, as the MetricsFeature adds itself before the Greetings Phase. and the scheduler starts after the greetings phase.